module-4/backend/week_one.md

```diff
@@ -1,19 +1,36 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+  I read most of Eloquent Javascript and did part of Wes Bos' JavaScript30 and that was pretty cool
 2. What are some tools to help debug JavaScript code?
+  We can use debugger, console log various things, use the source tab to see definitions of variables and our code file, and set breakpoints from the dev tools menu.
 3. What are some tools you need in order to unit test your JavaScript?
+  Mocha and Chai
 4. What is the syntax for invoking a function? Give an example.
+  var itsAFunction = function(args) { stuff happening(args)}
+
+  itsAFunction()
 5. What's `this` in JavaScript?
+  Everything in JavaScript is an object, more or less, and This refers to the specific object you're inside.
```

Comment:
Make sure you're able to go into a little more detail, give examples, etc. as you prep for interviews.